mailbox/tools/indexer/pom.xml

```diff
@@ -152,6 +152,10 @@
             <artifactId>testcontainers</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.apache.james</groupId>
```

Comment:
`${james.groupId}` ? and order it also :)

Reply:
Yes, thanks we saw this. It's in the do not review commit, and will be fixed with a rebase later on.
Nice catch :)